### PR TITLE
dx: create newsletter for January

### PR DESCRIPTION
**Pull request by jhchabran**
see title

### CODE REVIEW

File: content/departments/engineering/teams/dev-experience/newsletter.md [hunk 1]
```diff
@@ -8,6 +8,140 @@ To learn more about components of Sourcegraph's developer experience, check out
...
+
+**[An OTEL dashboard is now generated](https://github.com/sourcegraph/sourcegraph/pull/45009)**, allowing to monitor span emission/drops across any instance.
+
+**[Reduced the number of spans emitted by SQL requests](https://github.com/sourcegraph/sourcegraph/pull/44798)**.
+
+**[Updated docs for _tail sampling_](https://github.com/sourcegraph/sourcegraph/pull/44658)**.
+
+**[Upgraded `otel-collector` to `v0.65`](https://github.com/sourcegraph/sourcegraph/pull/44543)**, which allowed us to start using the `filter processor` to selectively send traces to different services depending on certain attributes. We're currently using this processor to send UI traces to a different dataset on Honeycomb!
+
+**[Deprecate use of Grafana cloud for tracing](https://github.com/sourcegraph/sourcegraph/issues/43935) as well as [exporting logs in CI](https://github.com/sourcegraph/sourcegraph/issues/43934)**
```
Comment (burmudar) [inline content/departments/engineering/teams/dev-experience/newsletter.md]:
```suggestion
**[Deprecate use of Grafana cloud for tracing](https://github.com/sourcegraph/sourcegraph/issues/43935) as well as [for exporting logs in CI](https://github.com/sourcegraph/sourcegraph/issues/43934)**
```

**Review by burmudar — APPROVED** [review]
Thanks for doing this! 🥇

**Review by sanderginn — APPROVED** [review]
long list yo

**Comment by kalanchan** [comment]
looks good!! :)

**Pull request merged**